code/modules/client/preference_setup/loadout/loadout_augments.dm

```diff
@@ -61,6 +61,19 @@
 	augs["retractable lighter, left hand"] = /obj/item/organ/internal/augment/tool/combitool/lighter/left
 	gear_tweaks += new /datum/gear_tweak/path(augs)
 
+/datum/gear/augment/tool/pen
```

Comment:
To follow the pattern of the other `/datum/gear/augment`s this should be `/datum/gear/augment/pen`.